curator-framework/src/main/java/org/apache/curator/framework/imps/ExistsBuilderImpl.java

```diff
@@ -185,7 +185,7 @@ public Stat forPath(String path) throws Exception
             OperationAndData<String> operationAndData = new OperationAndData<String>(this, path, backgrounding.getCallback(), null, backgrounding.getContext(), watching);
             if ( createParentContainersIfNeeded || createParentsIfNeeded )
             {
-                CreateBuilderImpl.backgroundCreateParentsThenNode(client, operationAndData, operationAndData.getData(), backgrounding, createParentContainersIfNeeded);
+                CreateBuilderImpl.backgroundCreateParentsThenNode(client, operationAndData, operationAndData.getData(), backgrounding, client.getAclProvider(), createParentContainersIfNeeded);
```

Comment:
I think yes, it'd better if one can set the ACL as well with the option of creating parents, because the ability to set the ACL when creating a node is the expected behaviour.